src/rollup/widgets/actions/construct-widget-actions.js

```diff
@@ -234,6 +241,9 @@ export default function constructWidgetActions (widget, config) {
           widgetOnButtonClick(event);
         }
 
+        // default to a three second delay if the user has not defined one
+        var delay = typeof config.success.delay !== 'undefined' ? config.success.delay * 1000 : 3000;
```

Comment:
This is reflected in the tests, but it looks like we're failing here if `config.success` is undefined. Lets try something like:

```
var delay = config.success && typeof config.success.delay !== 'undefined' ? config.success.delay * 1000 : 3000;
```

Reply:
Also just yesterday, we merged a PR: https://github.com/lytics/pathforajs/pull/256 that reorganizes a bunch of tests. So if you could pull the `develop` branch and merge it in to your branch, that will give you the most updated version of the tests. If you would like to run them locally, the command is:

```
gulp build && karma start --single-run --browsers PhantomJS
```